jetstream/resource_jetstream_stream.go

```diff
@@ -355,7 +360,7 @@ func resourceStreamRead(d *schema.ResourceData, m any) error {
 	d.Set("max_consumers", str.MaxConsumers())
 	d.Set("max_msgs", int(str.MaxMsgs()))
 	d.Set("max_msgs_per_subject", int(str.MaxMsgsPerSubject()))
-	d.Set("max_age", str.MaxAge().Seconds())
+	d.Set("max_age", int(str.MaxAge().Seconds()))
```

Comment:
hmm, as we used to force this down to -1 in old code we probably now have to keep doing that else old tf code will cause state changes for no actual change, like now I could put 0 here but old desired state would have been -1

Reply:
Leaving this comment here for posterity, but in the old code setting `max_age` to "-1" did nothing. `Set` returned an Error that was being ignored, but with the move to v2 of the SDK the code panics. This means that we always (correctly) set the value to 0 so removing the code here doesn't change the behaviour.